packages/ui-kit/src/components/multi-path-block.tsx

```diff
@@ -153,6 +153,7 @@ const SelectorsContainer = styled.div`
 const ComponentWrapper = styled.div`
   border: 2px solid ${designSystem.colors.light.borderPrimary};
   border-radius: ${designSystem.tokens.borderRadiusForMultiPath};
+  max-width: 100% !important;
```

Comment:
Just double checking if we can achieve max-width without !important?

Reply:
There's a generic rule that overrides it, I couldn't find a better way.
<img width="166" alt="Screenshot 2024-10-01 at 15 41 00" src="https://github.com/user-attachments/assets/2f378b98-779f-44e5-88d1-357bc0036ea3">